src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int bound = 100;
```

Comment:
Please, follow the rules of constants naming:
```suggestion
    private static final int BOUND = 100;
```